### PR TITLE
Windows memory dumps

### PR DESCRIPTION
**Pull request by ROCKNROLLKID**
This is https://github.com/az0/bleachbit/pull/86, except with the Winapp2 entries included. The other PR can be closed. The memory dump entry can be taken out of Winapp2, once this gets merged into CleanerML, as all the other cleaners have it in their own cleaning rules, outside of Winapp2.

This is my first ever trying to understand CleanerML, so there probably is a lot of mistakes.

### CODE REVIEW

**Comment by nodiscc** [comment]
Hi @ROCKNROLLKID, can you provde additional information as listed at https://github.com/az0/cleanerml#additional-information ? Specifically test results and specific targeted Windows versions.

Thanks

**Comment by ROCKNROLLKID** [comment]
This should work on any Windows OS, or at the very least Vista+.

I couldn't get it working, so I was hoping someone could help me as this is my first time ever trying to use CleanerML.

File: pending/memorydumps.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<cleaner id="memorydumps" os="windows">
+  <label>Windows Memory Dumps</label>
+  <description>Memory Dumps</description>
+  <option id="dumps">
+    <label>Memory Dumps</label>
+    <description>Deletes Windows memory dump files</description>
+    <action command="delete" search="file" path="$WINDIR/memory.dmp/
```
Comment (nodiscc) [inline pending/memorydumps.xml]: missing `>`

File: pending/memorydumps.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <action command="delete" search="glob" path="$WINDIR/MiniDump/*.dmp>
+    <action command="delete" search="glob" path="$APPDATA/Local/CrashDumps/*.dmp
```
Comment (nodiscc) [inline pending/memorydumps.xml]: missing `>` again

**Comment by nodiscc** [comment]
I do not have a windows machine at hand to test this. I have added 2 comments regarding missing closing `>`, maybe that's why you couldn't get it working?

**Comment by ROCKNROLLKID** [comment]
Still not working right. Getting this error:

**Comment by nodiscc** [comment]
It's still missing several closing `/` and `"`. Eg.

    <action command="delete" search="glob" path="$APPDATA/Local/CrashDumps/*.dmp>

should be
    <action command="delete" search="glob" path="$APPDATA/Local/CrashDumps/*.dmp"/>

**Comment by ROCKNROLLKID** [comment]
Well the errors are gone now, but I still can't get this to load on my copy of BleachBit.

**Pull request closed without merging**